backend/app/services/rag/retrieval_service.py

```
@@ -714,6 +754,180 @@ async def _execute_runtime_query(
+            query = (
+                db.query(
+                    KnowledgeDocument.id,
+                    KnowledgeDocument.kind_id,
+                    SubtaskContext.text_length,
+                )
+                .select_from(KnowledgeDocument)
+                .join(
+                    SubtaskContext,
+                    KnowledgeDocument.attachment_id == SubtaskContext.id,
+                )
+                .filter(KnowledgeDocument.is_active == True)
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `True`; use `KnowledgeDocument.is_active:` for truth checks

Replace with `KnowledgeDocument.is_active`

(E712)